tests/TestFormula.elm

```diff
@@ -3,9 +3,12 @@ module TestFormula exposing (..)
 import Dict
 import Expect
 import Formula exposing (..)
```

Comment:
It should be sufficient to expose just the type and its constructors here:
`import Formula exposing (Formula(..))`
Functions from the `Formula` module should always be used qualified.